<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LaserDockWidget</class>
 <widget class="QWidget" name="LaserDockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>160</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>330</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Laser 1</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>311</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="layoutLaser1">
     <item>
      <widget class="Led" name="laser1Led">
       <property name="minimumSize">
        <size>
         <width>15</width>
         <height>15</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>15</width>
         <height>15</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../ressources.qrc">:/img/img/led_off_red.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="onOffLaser1">
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>On</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="laser1Spin">
       <property name="suffix">
        <string>%</string>
       </property>
       <property name="prefix">
        <string/>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="laser1Slider">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="tracking">
        <bool>false</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>0</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../ressources.qrc">:/img/img/laser.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>330</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Laser 2</string>
   </property>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>311</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="layoutLaser2">
     <item>
      <widget class="Led" name="laser2Led">
       <property name="minimumSize">
        <size>
         <width>15</width>
         <height>15</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>15</width>
         <height>15</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../ressources.qrc">:/img/img/led_off_red.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="onOffLaser2">
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>On</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="laser2Spin">
       <property name="suffix">
        <string>%</string>
       </property>
       <property name="prefix">
        <string/>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="laser2Slider">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="tracking">
        <bool>false</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>0</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../ressources.qrc">:/img/img/laser.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Led</class>
   <extends>QLabel</extends>
   <header>Led.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../ressources.qrc"/>
 </resources>
 <connections/>
</ui>
